io.github.wojciech_graj.OrbVis.yml

```diff
@@ -0,0 +1,29 @@
+app-id: io.github.wojciech_graj.OrbVis
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: orbvis
+finish-args:
+  - --share=ipc
+  - --socket=fallback-x11
+  - --share=network
+  - --filesystem=xdg-documents/io/github/wojciech_graj/OrbVis/
```

Comment:
you mean it will create `io/github/wojciech_graj/OrbVis/` in `Documents`?

Reply:
Yes, that was my intention. Is there an issue with this?